packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -176,6 +176,13 @@ export class Tooltip {
     e.stopPropagation();
   }
 
+  @Listen('keydown', { target: 'document' })
```

Comment:
Dit lijkt me niet zo efficient, nu hebben we per tooltip een keydown listener op `document`. Het lijkt me logischer pas te luisteren zodra de tooltip toont.